src/pydata_sphinx_theme/assets/styles/content/_tables.scss

```diff
@@ -5,8 +5,6 @@
 // ensure table will fit in the article width and make them y-scrollable
 table {
   display: table;
-  width: fit-content;
-  max-width: 100%;
```

Comment:
I was curious about this CSS change, so I built the docs from this PR and inspected them locally using the following commands:

```
stb compile
sphinx-build docs docs/_build/html
python -m http.server -d docs/_build/html
```

I then opened the `docs/_build/html` in VS Code and searched all files for "<table" to find pages in our docs with HTML tables. I compared the tables on the following pages with the ones in production and couldn't detect any major differences. I've noted small differences where I found them:

- [/genindex.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/py-modindex.html)
- [/py-modindex.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/py-modindex.html)
- [/api/pydata_sphinx_theme/index.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/api/pydata_sphinx_theme/index.html) 
- plus 2 other pages under /api/pydata_sphinx_theme/ 👍 
- [/examples/pydata.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/pydata.html) - the only difference is that on the live site the font in data table is bigger, which is more accessible and looks better IMHO
- [/examples/kitchen-sink/api.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/kitchen-sink/api.html)
- [/examples/kitchen-sink/images.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/kitchen-sink/images.html)
- [/examples/kitchen-sink/lists.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/kitchen-sink/lists.html)
- [/examples/kitchen-sink/tables.html](https://pydata-sphinx-theme.readthedocs.io/en/latest/examples/kitchen-sink/tables.html)

So it looks like this change is safe. I think the rationale for the change is that the nbsphinx provides its own table styles, and these rules collide with those. 

To be honest the `width: fit-content` rule doesn't make sense to me anyway, given the comment directly above `table` selector.

Reply:
I remember adding it for something else but I think it's a legacy from a previous .css refactoring